roles/check_secrets/tasks/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+- name: Check secrets
+  fail:
+    msg: "ERROR: It looks like your secrets.yml file is missing..."
```

Comment:
We could be nice to users and explain how to provide them. I would add something like "specify them using providing the argument '-e @path/to/secrets.yml'"

Reply:
Done.